components/MeetingDetail/MeetingDetail.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import DogName from "../DogName/DogName";
+import ImageWithText from "../ImageWithText/ImageWithText";
+
+export default function MeetingDetail({ data }) {
+  const { location, date, time, excluded, furtherInfo, attending } = data;
+  return (
+    <StyledArticle>
+      <h2>{location}</h2>
+      <ImageWithText
+        image="calendar-icon.svg"
+        altText="Kalender Icon"
+        text={date}
+      />
+      <ImageWithText image="clock-icon.svg" altText="Uhr Icon" text={time} />
+      {excluded ? (
+        <ImageWithText
+          data-testid="excluded"
+          image="stop-icon.svg"
+          altText="Stop Icon"
+          text={excluded}
+        />
+      ) : null}
+      <StyledInfobox>
+        {furtherInfo ? (
+          <div data-testid="info">
+            <label htmlFor="weitereInfos">Weitere Infos:</label>
+            <StyledParagraph id="weitereInfos">{furtherInfo}</StyledParagraph>
+          </div>
+        ) : null}
+      </StyledInfobox>
+      <StyledInfobox>
+        <label htmlFor="teilnehmende">Wir sind dabei:</label>
+        <DogName id="teilnehmende" attendingDog={attending} />
+      </StyledInfobox>
+    </StyledArticle>
+  );
+}
+
+const StyledArticle = styled.article`
```

Comment:
Would it be better to use "section"or "main" here? I saw that you use "section" in DogName.  A section element in a div element. the div element is in a article element, might be not so right?

Reply:
Thanks for the comment. I changed it to main and will keep in mind to change it back to article if the structure of this page changes and this will only be a smaller part of it.